infra/project-config/main.tf

```diff
@@ -3,13 +3,15 @@ locals {
   # This will be used in names of AWS resources
   project_name = "<PROJECT_NAME>"
 
-  # Project owner
+  # Project owner (e.g. navapbc)
```

Comment:
can we add another line to the comment saying something along the lines of "this will be used to tag infrastructure resources"